client/slack.go

```diff
@@ -208,3 +222,34 @@ func newUser(u *slack.User) user {
 		LastName:  u.Profile.LastName,
 	}
 }
+
+func newReactionEvent(e *slack.ReactionAddedEvent, u *slack.User, itemuser *slack.User) reactionAddedEvent {
+	return reactionAddedEvent{
+		ReactionUser:   newUser(u),
+		ReactionName:   e.Reaction,
+		EventTimestamp: e.EventTimestamp,
+		ItemTimestamp:  e.Item.Timestamp,
+		ItemType:       e.Item.Type,
+		ChannelId:      e.Item.Channel,
+		ItemUser:       newUser(itemuser),
+	}
+
+}
+
+type reactionAddedEvent struct {
+	ReactionUser   user   `json:"user"`
+	ReactionName   string `json:"reaction"`
+	EventTimestamp string `json:"eventTimestamp"`
+	ItemType       string `json:"type"`
+	ItemTimestamp  string `json:"itemTimestamp"`
+	ItemUser       user   `json:"itemUser"`
+	ChannelId      string `json:"channelId"`
+}
```

Comment:
replacing previous suggestions with one this is for consistency with the underlying event. I believe you've missed type event and this is what you are trying to log above

```suggestion
type reactionItem struct {
	Type      string `json:"type"`
	Timestamp string `json:"timestamp"`
	User      user   `json:"user"`
	Channel   string `json:"channel"`
}

type reactionEvent struct {
	Type           string       `json:"type"`
	User           string       `json:"user"`
	ItemUser       string       `json:"itemUser"`
	Item           reactionItem `json:"item"`
	Reaction       string       `json:"reaction"`
	EventTimestamp string       `json:"eventTimestamp"`
}
```